prime-router/src/main/kotlin/fhirengine/translation/HL7toFhirTranslator.kt

```diff
@@ -104,4 +105,12 @@ class HL7toFhirTranslator internal constructor(
         bundle.identifier.value = mshSegment.messageControlID.value
         bundle.identifier.system = "https://reportstream.cdc.gov/prime-router"
     }
+
+    /**
+     * Enhance the [bundle] Provenance to include references to diagnostic reports.
+     */
+    private fun enhanceBundleProvenance(bundle: Bundle) {
+        // bundles must contain target references to diagnostic reports
```

Comment:
This function seems unnecessary to me since it's only calling another function.

Reply:
I removed the function and called the helper directly. We can explore changing this if we ever need to do more to Provenance at this stage in the future.